### ISSUE TITLE
Auth Provider migration fails in MsSQL

### ISSUE THREAD

**Issue by w7shdev** [directus/directus#8499]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

When `create-directus-project` it breaks if MsSQL database was selected because of `Auth Provider migration fails` and you can not complete the installation. 

**Here a screenshot**
<img width="1382" alt="Screen Shot 2021-10-01 at 10 07 12 PM" src="https://user-images.githubusercontent.com/60617324/135683116-8bbd31dd-56c5-41e1-89c7-9841e4c78745.png">

I copy paste the SQL statement and paste it in Azure cloud studio 
```
ALTER TABLE [directus_users] ALTER COLUMN [email] nvarchar(128) null
```

<img width="588" alt="Screen Shot 2021-10-02 at 12 36 01 AM" src="https://user-images.githubusercontent.com/60617324/135683269-0450bb49-24f1-4196-a449-97c6f9f08d30.png">

so I went to directus updated the migration file and did something like this 

```JS
//in dist/database/migrations/20210903A-add-auth-provider.js
 await knex.schema.alterTable('directus_users', (table) => {
        table.string('provider', 128).notNullable().defaultTo('default');
        table.string('external_identifier').unique();
        table.dropUnique('email'); 
    });
    await knex.schema.alterTable('directus_users', (table) => {
        table.string('email', 128).nullable().unique().alter(); 
    });
```

and then I did `npx directus init` and the installation completed 

### To Reproduce

1. Create new directus project
2. use Microsoft SQL server 
 

### What version of Directus are you using?

v9.0.0-rc.95

### What version of Node.js are you using?

v14.16.1

### What database are you using?

Microsoft SQL Server 2017 14.0.3029.16 (X64) 	

### What browser are you using?

N/A

### What operating system are you using?

MacOs 

### How are you deploying Directus?

running locally

**Comment by aidenfoxx** [comment]
This is a duplicate of #8368 and has been fixed in #8399.

**Comment by w7shdev** [comment]
> This is a duplicate of #8368 and has been fixed in #8399.

My bad, I should have searched within closed issues -- the current latest `create-directus-projects` works with MySQL this is just with MsSQL it fails

**Comment by w7shdev** [comment]
I'll close this issue
